### ISSUE TITLE
Zeal tab should not show "Loading..." when Zeal is started

### ISSUE THREAD

**Issue by sanel** [zealdocs/zeal#1640]
When you start Zeal, it will show "Loading..." in tab, but nothing will be loaded in the main view (as shown in the screenshot). IMHO, it should either load the first item from the list (that can be slow) or should show empty tab (or no tabs at all), before the first item is selected.

![2024-07-08_12-21](https://github.com/zealdocs/zeal/assets/213914/94eb19e5-79c3-4ad9-897e-dc18d2206851)

**Comment by meoowth** [comment]
me too. do u work out?

**Comment by trollixx** [comment]
What is your Zeal version and OS?

**Comment by sanel** [comment]
Tested with 0.7.1 and the latest git version. OS is Slackware 15.0

**Comment by sanel** [comment]
Looks like "Loading..." is added by [default].(https://github.com/zealdocs/zeal/blob/main/src/libs/ui/mainwindow.cpp#L335). I think "Welcome" was before, can't recall 100%.

**Comment by trollixx** [comment]
Are you seeing an empty page? That means the home page (with Welcome title) does not load. Anything in the console output?

**Comment by sanel** [comment]
Yes, I see empty page. Here is the output from zeal git version:

```
$ ./build/bin/zeal 
zeal.core.applicationsingleton: Singleton ID: va1X0JUe8bmvHzhlZMk_a2Cv-V0mauKmu5lwiW19DTs
zeal.core.applicationsingleton: Starting as a primary instance. (PID: 25377)
QVariant::load: unable to load type 1032.
zeal.core.httpserver: Listening on http://127.0.0.1:43169...
zeal.core.httpserver: Mounted '/home/foo/.local/share/Zeal/Zeal/docsets/Ansible.docset/Contents/Resources/Documents' to '/ansible'.
zeal.core.httpserver: Mounted '/home/foo/.local/share/Zeal/Zeal/docsets/Bash.docset/Contents/Resources/Documents' to '/bash'.
...
```

**Comment by trollixx** [comment]
Oh, I think I figured it out. This is happening because Qt WebEngine process crashes. So, the tab never receives the load finished signal and gets stuck.

**Comment by trollixx** [comment]
#1896 is addressing the bad UX and adds a few attempts to restart the renderer when it crashes, but the actual fix/workaround will be provided via #1692 and #1769.

**Comment by sanel** [comment]
Tested with the latest Zeal source and it works, thanks!

<img width="1213" height="1084" alt="Image" src="https://github.com/user-attachments/assets/d5f92c6b-aa22-4bea-9374-7cf0702bf04b" />
